### DISCUSSION TITLE
Requesting support for Magma

### DISCUSSION THREAD

**Discussion by JustRed23** [dmulloy2/ProtocolLib#1729] [Ideas]
Hello there, Magma dev here,

People that use our service have requested that we support protocollib for a long time, i would like to help you provide an interface for protocollib to work with if you are willing to.

If you didn't know, Magma is a forge/craftbukkit hybrid making it possible to run plugins on a forge server, we currently support 1.12.2, 1.16.5 and 1.18.2 (this version is still in development and highly unstable).

I would like to hear from you regarding this.
JustRed23

**Comment by derklaro** [comment]
Just use correct spigot mappings, then there is no issue 🤷. But supporting or maintaining different mappings versions is not a thing that we want to do imho.

**Reply by JustRed23** [reply]
Yea seems fair, thanks for answering though.
